draft-ietf-webtrans-overview.md

```diff
@@ -260,7 +300,67 @@ Streams SHOULD be sufficiently lightweight that they can be used as messages.
 Data sent on a stream is flow controlled by the transport protocol.  In addition
 to flow controlling stream data, the creation of new streams is flow controlled
 as well: an endpoint may only open a limited number of streams until the peer
-explicitly allows creating more streams.
+explicitly allows creating more streams.  From the perspective of the client,
+this is presented as a size-bounded queue of incoming streams.
+
+Any WebTransport protocol SHALL provide the following operations on the
+session:
+
+create a unidirectional stream
+: Creates an outgoing unidirectional stream; this operation may block until the
+  flow control allows for it to be completed.
+
+create a bidirectional stream
+: Creates an outgoing bidirectional stream; this operation may block until the
+  flow control allows for it to be completed.
```

Comment:
I think this identifies a problem with overloading the term flow control to apply to actual data and stream counts. I can't disambiguation which one you mean here, or could it be both?

Reply:
I think it should be only stream-level flow control, but here it's just "any form of flow control that applies".

Reply:
Perhaps saying "flow control of the underlying protocol"?